### PR TITLE
[MINOR][DOCS] Fix typos at python/pyspark/sql/types.py 

### PR DESCRIPTION
**Pull request by sharifahmad2061**
### What changes were proposed in this pull request?

This PR fixes some typos in `python/pyspark/sql/types.py` file.

### Why are the changes needed?

To deliver correct wording in documentation and codes.

### Does this PR introduce any user-facing change?

Yes, it fixes some typos in user-facing API documentation.

### How was this patch tested?

Locally tested the linter.

### CODE REVIEW

**Comment by HyukjinKwon** [comment]
@sharifahmad2061, that's fine but can you check if there are more typos in this file while we're here? I am sure there are more.

**Comment by sharifahmad2061** [comment]
@HyukjinKwon Yes there are some other grammatical mistakes. I found them but thought some other newbie should clean his hands on it. Should I correct them all and commit them?

**Comment by HyukjinKwon** [comment]
Yeah, let's kind them together while we're here. It's not worth making PRs for each typo.

**Comment by sharifahmad2061** [comment]
@HyukjinKwon can you review the changes in the new commit.

**Comment by HyukjinKwon** [comment]
ok to test

**Comment by sharifahmad2061** [comment]
@HyukjinKwon there was a small linting issue and I corrected it locally. Should I upload it to my master.

**Comment by HyukjinKwon** [comment]
No problem. Just push some changes into the branch where your PR was open against. Seems it's your `master` branch.

**Comment by HyukjinKwon** [comment]
@shagunsodhani, I fixed PR title and description for you this time. Next time, please refer the correct form and review https://spark.apache.org/contributing.html.

File: python/pyspark/sql/types.py [hunk 1]
```diff
@@ -1433,7 +1433,7 @@ class Row(tuple):
     ``key in row`` will search through row keys.
 
     Row can be used to create a row object by using named arguments.
-    It is not allowed to omit a named argument to represent the value is
+    It is not allowed to omit a named argument to represent that the value is
```
Comment (HyukjinKwon) [inline python/pyspark/sql/types.py]: I think it was correct to omit `that` as well.

Comment (sharifahmad2061) [reply python/pyspark/sql/types.py]: yes, we can omit `that`, but adding it makes the sentence a little bit more explanatory.

**Comment by HyukjinKwon** [comment]
@shagunsodhani, can you push your changes to your master branch?

**Comment by sharifahmad2061** [comment]
@HyukjinKwon I have pushed all my changes to my master branch.

**Review by HyukjinKwon — APPROVED** [review]

**Comment by sharifahmad2061** [comment]
Thanks @HyukjinKwon

**Comment by HyukjinKwon** [comment]
Merged to master and branch-3.0.

File: python/pyspark/sql/types.py (hunk 1 above)
Comment (HyukjinKwon) [reply python/pyspark/sql/types.py]: @sharifahmad2061 I just merged this time but let's don't make unnecessary changes like this next time.

**Pull request closed without merging**